main/test/ca/uwaterloo/flix/library/TestMutList.flix

```diff
@@ -1200,109 +1207,34 @@ namespace TestMutList {
     @test
     def compress!01(): Bool & Impure =
         let v = MutList.range(0, 100);
-        List.foreach(_ -> let _ = MutList.pop!(v); (), List.range(0, 75));
-        let c = MutList.capacity(v);
+        let f = _ -> { let _ = MutList.pop!(v); () };
```

Comment:
Since this is a bit more logic than is typical in a test, can you add a quick comment to explain what's going on here?

Reply:
Yeah sure, I had a lot of trouble understanding it myself too before I refactored it a bit. Should be better in the upcoming commits.